dags/bqetl_ctxsvc_derived.py

```diff
@@ -82,6 +82,22 @@
         arguments=["--schema_update_option=ALLOW_FIELD_ADDITION"],
     )
 
+    contextual_services_derived__event_aggregates_check__v1 = bigquery_etl_query(
```

Comment:
We had the incident because two tasks were in the same DAG and one affected the other. Does putting the check also in the same DAG make the check vulnerable to the same issues?

Reply:
I don't think it will... The check is configured to run after the ETL for processing these events has completed.